### ISSUE TITLE
when I resume run I miss some training data like pr_curve, confusion matrix, how to get them

### ISSUE THREAD

**Issue by besbesmany** [ultralytics/yolov5#4540]
when I use resume in colab

`!python train.py --resume wandb-artifact://{crashed_run_path}`

I lost many files after the run ends like
P_curve, Pr_curve, confusion matrix, results.png, R_curve 

how to get them??

![50](https://user-images.githubusercontent.com/35992924/130783351-1e4926fe-2dda-42b3-a92e-19cca1c46a9a.png)
![60](https://user-images.githubusercontent.com/35992924/130783359-15af51a8-bc55-42f5-9ae6-f0974c25c4b6.png)

**Comment by glenn-jocher** [comment]
@besbesmany 👋 Hello! Thanks for asking about **resuming training**. YOLOv5 🚀 Learning Rate (LR) [schedulers](https://github.com/ultralytics/yolov5/blob/9f3a388cea66ca71b9bbe057929eed8fbc920a7e/train.py#L109-L142) follow predefined LR curves for the fixed number of `--epochs` defined at training start (`default=300`), and are designed to fall to a minimum LR on the final epoch for best training results. For this reason you can not modify the number of epochs once training has started.

<img width="850" alt="LR Curves" src="https://user-images.githubusercontent.com/26833433/119332195-8e113a80-bc88-11eb-8eb7-7494ff1d5c01.png">

If your training was **interrupted** for any reason you may continue where you left off using the `--resume` argument. If your training fully completed, you can start a **new** training from any model using the `--weights` argument. Examples:

## Resume Single-GPU

You may not change settings when resuming, and **no additional arguments** other than `--resume` should be passed, with an optional path to the checkpoint you'd like to resume from. If no checkpoint is passed the most recently updated `last.pt` in your `yolov5/` directory is automatically found and used:
```bash
python train.py --resume  # automatically find latest checkpoint (searches yolov5/ directory)
python train.py --resume path/to/last.pt  # specify resume checkpoint
```

## Resume Multi-GPU

[Multi-GPU DDP trainings](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training) must be resumed with the same GPUs and DDP command, i.e. assuming 8 GPUs:
```bash
python -m torch.distributed.run --nproc_per_node 8 train.py --resume  # resume latest checkpoint
python -m torch.distributed.run --nproc_per_node 8 train.py --resume path/to/last.pt  # specify resume checkpoint
```

## Start from Pretrained

If you would like to **start training** from a fully trained model, use the `--weights` argument, not the `--resume` argument:
```bash
python train.py --weights path/to/best.pt  # start from pretrained model
```

Good luck and let us know if you have any other questions!

**Comment by besbesmany** [comment]
I resume using this line

`!python train.py --resume wandb-artifact://{crashed_run_path}`

the output files are not exist in resuming case after the run is finished

how to get these files??

P_curve, Pr_curve, confusion matrix, results.png, R_curve??

**Comment by glenn-jocher** [comment]
@besbesmany all the output results should be there if you resume or if you train in one go. If you can reproduce the issue please submit a reproducible example to help us get started debugging it, thank you!

**Comment by besbesmany** [comment]
I don't get them when I use  wandb-artifact://{crashed_run_path}

**Comment by glenn-jocher** [comment]
@AyushExel can you take a look at this? 

@besbesmany is saying that resuming from a W&B artifact is not producing all the output metrics and plots.

@besbesmany does this occur when resuming locally or only when resuming from a W&B artifact?

**Comment by AyushExel** [comment]
@besbesmany can you share the run link of the run that you resumed?

**Comment by AyushExel** [comment]
@besbesmany I couldn't reproduce the problem. See[ this run ](https://wandb.ai/cayush/yoloV5/runs/c61mohyk?workspace=)that I resumed from artifact. It has all the metrics under the "Results" panel

**Comment by besbesmany** [comment]
yes they appears in wandb but not in train folder of colab 

thank you
